src/main/java/com/testmonitor/resources/TestCaseFolder.java

```diff
@@ -0,0 +1,141 @@
+package com.testmonitor.resources;
+
+import org.apache.hc.core5.http.NameValuePair;
+import org.apache.hc.core5.http.message.BasicNameValuePair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestCaseFolder {
```

Comment:
A constant defining root would be nice?

Reply:
I added it on the actions

Reply:
Actions... is a bit weird? It isn't an action?

TestCaseFolder.root(projectId) makes more sense?